EncounterMe/EncounterMe/Views/MapPage.xaml.cs

```diff
@@ -443,18 +448,19 @@ private void Button_Type_Car(object sender, EventArgs e)
         //could be better, if the line was dotted
         private void UserLocationChangedEventHandler(Location currentLocation)
         {
+            //Calculating the difference of last registered location and current location
             double distance = Location.CalculateDistance(_lastRegisteredLocation.Latitude, _lastRegisteredLocation.Longitude,
                                                                 currentLocation.Latitude, currentLocation.Longitude, DistanceUnits.Kilometers);
 
-            //Calculates offset
+            //Calculates distance offset (used for further calculations)
             double distanceOffset = 0.1 * _averageDistance;     //TODO : check what works with offset percentage
 
-            if (distance != 0 && _averageDistance != 0)
+            if (distance > 0 && _averageDistance > 0)
             {
                 if (distance < _averageDistance + distanceOffset)
                 {
-                    _lastRegisteredLocation = currentLocation;
-                    RedrawFirstPolyline(currentLocation);
+                    //_lastRegisteredLocation = currentLocation;
```

Comment:
O kodėl šitie nebereikalingi? Gal tada reikėtų pakeist sąlyga, kad tai kas 'else' būtų vykdoma čia, nes truputį keistai atrodo if be jokio vykdymo

Reply:
Nes tipo, negraziai atrodo. As turejau dar susiplanaves padaryt dalyku, bet bsk nepavyko, tai sita koda palikau, tsg uzkomentavau

Reply:
Galima ir atkomentuot, ne problema